entity-service-impl/src/main/java/org/hypertrace/entity/data/service/DocumentParser.java

```diff
@@ -10,7 +10,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-class DocumentParser {
+public class DocumentParser {
```

Comment:
Why is it to change to `public`?

Reply:
To be able to use the method `parseOrLog`

Reply:
It was already used inside `searchByQueryAndStreamResponse` prior.